xknx/devices/light.py

```diff
@@ -34,6 +34,57 @@ class ColorTempModes(Enum):
     relative = "DPT-5.001"
 
 
+class _SwitchAndBrightness:
+    def __init__(
+        self,
+        xknx,
+        name,
+        feature_name: str,
+        group_address_switch=None,
+        group_address_switch_state=None,
+        group_address_brightness=None,
+        group_address_brightness_state=None,
+        after_update_cb=None,
+    ):
+        self.switch = RemoteValueSwitch(
+            xknx,
+            group_address_switch,
+            group_address_switch_state,
+            device_name=name,
+            feature_name=feature_name + "_state",
+            after_update_cb=after_update_cb,
+        )
+        self.brightness = RemoteValueScaling(
+            xknx,
+            group_address_brightness,
+            group_address_brightness_state,
+            device_name=name,
+            feature_name=feature_name + "_brightness",
+            after_update_cb=after_update_cb,
+            range_from=0,
+            range_to=255,
+        )
+
+    @property
+    def is_on(self):
+        """Return if light supports color."""
```

Comment:
```suggestion
        """Return if light is on."""
```

Reply:
thanks for your review, this must have been an error on my side
Fixed in https://github.com/XKNX/xknx/pull/508/commits/2480c5bf035195e15d9bac4c754415161e5621ce